### PR TITLE
Change the group ID and rename the sub modules

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
## Purpose
Related to ballerina-platform/ballerina-standard-library#1623

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/Ballerina.toml [hunk 1]
```diff
@@ -4,7 +4,5 @@ name = "random"
 version = "@toml.version@"
 
 [[platform.java11.dependency]]
-path = "../random-native/build/libs/random-native-@project.version@.jar"
-groupId = "ballerina"
-artifactId = "random"
+path = "../native/build/libs/random-native-@project.version@.jar"
 version = "@project.version@"
```
Comment (kalaiyarasiganeshalingam) [inline ballerina/Ballerina.toml:8]: No need version too

File: native/build.gradle [hunk 2]
```diff
@@ -70,8 +70,7 @@ tasks.withType(Checkstyle) {
 }
 
 spotbugsMain.finalizedBy validateSpotbugs
-checkstyleMain.dependsOn(":build-config:checkstyle:downloadMultipleFiles")
-
+checkstyleMain.dependsOn(":checkstyle:downloadMultipleFiles")
```
Comment (ThisaruGuruge) [inline native/build.gradle:73]: Shall we change this task to `downloadCheckstyleRuleFiles`? 

```suggestion
checkstyleMain.dependsOn(":checkstyle:downloadMultipleFiles")
```

File: ballerina/Ballerina.toml (hunk 1 above)
Comment (MadhukaHarith92) [reply ballerina/Ballerina.toml]: Updated in 73efe20

File: native/build.gradle (hunk 2 above)
Comment (MadhukaHarith92) [reply native/build.gradle]: Updated in 73efe20

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**